ui/app/utils/build-api-validators.js

```diff
@@ -0,0 +1,20 @@
+import { validator, buildValidations } from 'ember-cp-validations';
```

Comment:
Can you add some comments explaining what this util does?  A good example of this is in the uitls/api-path.js file.  Something that would help a future reader understand its purpose and usage.